### ISSUE TITLE
MedRequest.status definition

### ISSUE THREAD

**Issue by PeteSalis** [nhsconnect/gpconnect#1058]
Is the way that TPP have implemented this field correct?

Is our spec correct. This field should be the status of the medication order and not a reflection of whether the patient is taking the medication.

**Comment by PeteSalis** [comment]
Also check the consumer guidance.

**Comment by martinhillyard** [comment]
Consumer data may need to be updated. Needs to be checked along with the guidance.

**Comment by PeteSalis** [comment]
Discussed with Martin. 

We agree that for Med Request status where the intent is order it should not be active. 

Where the intent is plan it is complete once all the authorised issues have been made or the authorisationExpiryDate has been reached. For repeat dispense once the issues have been made then the plan is complete

I will write the new versionof the MedRequest.status

For Medication Statement we agreed that we need to run the FHIR definition past the clinicians.

This could be used to carry the status from the system. This would mean this field in GP Connect is populated inconsistently as that is the behaviour in the GP estate. If we followed the exact FHIR definition it would give consumers the option of using the more consistent MedRequest.status or the variable MedStatement.status. Is this desirable or will it just be more confusing for consuming systems?

**Comment by martinhillyard** [comment]
Change applied. Release note required

**Comment by martinhillyard** [comment]
Typo for completed code needs to be changed

**Comment by martinhillyard** [comment]
Changes don't seem to have not been merged into 1.6.2 as yet. Fix version needs to be updated according to where these changes will be applied or issue closed if the changes are not going to be made.
